src/parser/smt2/smt2.cpp

```diff
@@ -1399,9 +1399,9 @@ void Smt2::notifyNamedExpression(cvc5::Term& expr, std::string name)
     parseError(
         "Cannot name a term in a binder (e.g., quantifiers, definitions)");
   }
-  // Note that we do not bind the symbol here; this is done separately
-  // in a define-fun command in Smt.g to ensure -o raw-benchmark results in a
-  // parsable result.
+  // define the variable. This needs to be done here so that in the rest of the
+  // command we can use this name, which is required by the semantics of :named.
```

Comment:
Can you further comment that because we bind the symbol tot he definition, the output of `-o raw-benchmark` and proofs does not consider names?

Reply:
will do